src/main/kotlin/net/irisshaders/lilybot/extensions/moderation/TerminalModeration.kt

```diff
@@ -354,7 +361,14 @@ class TerminalModeration : Extension() {
 		/** The ID of the user to unban. */
 		val userArgument by user {
 			name = "unbanUserId"
-			description = "Person Unbanned"
+			description = "Person to un-ban"
+		}
+
+		/** The reason for the un-ban. */
+		val reason by defaultingString {
+			name = "reason"
+			description = "The reason for the un-ban"
+			defaultValue = "No Reason Provided"
```

Comment:
Why is this all caps? Why are they all like this?

Reply:
Why not. All fixed